### PR TITLE
Bump babel and postcss

### PR DESCRIPTION
**Pull request by rebornix**

### CODE REVIEW

**Review by DonJayamanne — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -169,6 +169,7 @@
         "@phosphor/widgets": "^1.9.3",
         "@vscode/jupyter-ipywidgets7": "^2.0.1",
         "@vscode/jupyter-ipywidgets8": "^1.0.2",
+        "canvas": "^2.11.2",
```
Comment (rebornix) [inline package.json:172]: @DonJayamanne  It won't compile if `canvas` is still marked as optional on Linux it seems.

Comment (DonJayamanne) [reply package.json:172]: We shoudln't addi it as a `dependency` as its definitely not a dependency, perhaps dev dependency, 
However from the dependency tree this is not at all used.

Please remove  `canvas`  completely
Its not used, I checked,
I moved it from `devDependencies` to `optional` in the past, but its never been used.
Perhaps I removed something subsequently that requried canvas, e.g. some tests.

**Review by DonJayamanne — CHANGES_REQUESTED** [review]
Please remove `canvas`

File: package.json (hunk 1 above)
Comment (rebornix) [reply package.json:172]: @DonJayamanne unfortunately vega depends on canvas

```
ERROR in ./node_modules/@nteract/any-vega/vg2vl1.js 49651:17-34
Module not found: Error: Can't resolve 'canvas' in '/workspaces/vscode-notebook-renderers/node_modules/@nteract/any-vega'
resolve 'canvas' in '/workspaces/vscode-notebook-renderers/node_modules/@nteract/any-vega'
  Parsed request is a module
  using description file: /workspaces/vscode-notebook-renderers/node_modules/@nteract/any-vega/package.json (relative path: .)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
```

**Review by DonJayamanne — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (DonJayamanne) [reply package.json:172]: Oh, so it was never a dev dependency then.
also thats weird, as its not marked as a dependency at all.
probably used in code but not defined in their package.json

Comment (DonJayamanne) [reply package.json:172]: I'm going to fix this in debt week, we do not need the whole canvas package
It compiles native modules.
All we need is just this one simple JS file as everything runs in the browser
https://github.com/Automattic/node-canvas/blob/master/browser.js

Comment (DonJayamanne) [reply package.json:172]: & yes, its a dependency thats not defined in their package.json

**Pull request merged**